### PR TITLE
Create logo picture and text for header.

### PR DESCRIPTION
**Pull request by simtimj**

### CODE REVIEW

**Comment by simtimj** [comment]
@richchuynh Please review my pull request when you get the chance. Thanks!

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  render() {
+    return (
+      <div>
+        <Header />
+      </div>
+    )
+  }
+}
+
+export default App;
```
Comment (richchuynh) [inline client/src/components/App.jsx]: If following AirBnB style, we will need to create a new line entry for our last line of code.

File: client/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App.jsx'
+
+ReactDOM.render(<App />, document.getElementById('app'));
```
Comment (richchuynh) [inline client/src/index.jsx]: If following AirBnB style, we will need to create a new line entry for our last line of code.

File: client/src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+  render() {
+    return (
+      <div id='logo'>
+        {/* Logo and 'Hostelworld'*/}
+        <img id='logoPic' src="https://www.complaintsboard.com/thumb.php?src=hostelworld.png&wmax=182&hmax=300&quality=70&nocrop=1&bname=121731" height="50" width="50"/>
```
Comment (richchuynh) [inline client/src/components/Header.jsx]: AirBnB guide suggests single quotes. Can you refactor the other attributes in this line?

File: client/src/components/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        <h2 id='logoText' >HOSTELWORLD</h2>
```
Comment (richchuynh) [inline client/src/components/Header.jsx]: There seems to be an extra whitespace after 'logoText'

**Pull request merged**